### DISCUSSION TITLE
API about

### DISCUSSION THREAD

**Discussion by Hourunze1997** [FreshRSS/FreshRSS#7111] [Q&A]
/p/i/?a=rss&user=&token=&hours=168&nb=10&order=DESC
When I use this api to get the news from the feed, I want to know if there are fields in the returned results that can let me know the feed of the message and the category to which the feed belongs. Thank you.

**Comment by Alkarex** [comment]
Hello,

First, the URL you show is not really an API but "just" an RSS output. For better functionalities including feed and category information, check our GReader compatible API:
https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html#google-reader-compatible-api

Second, the URL you show is not the recommended one anymore. See instead:
https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries

Third, our RSS outputs do not contain provenance information at the moment besides the articles links, but we could consider adding for instance the original feed. Suggestions of Atom / RSS ontologies welcome to express that.
